### PR TITLE
Fix some untyped member serialisation

### PR DESCRIPTION
**Pull request by azhirnov**
Small improvement for structure member representation.

### CODE REVIEW

File: renderdoc/driver/vulkan/vk_common.h [hunk 1]
```diff
@@ -1091,6 +1091,7 @@ DECLARE_REFLECTION_ENUM(VkComponentSwizzle);
 DECLARE_REFLECTION_ENUM(VkCompositeAlphaFlagBitsKHR);
 DECLARE_REFLECTION_ENUM(VkConservativeRasterizationModeEXT);
 DECLARE_REFLECTION_ENUM(VkCullModeFlagBits);
+DECLARE_REFLECTION_ENUM(VkColorComponentFlagBits);
```
Comment (baldurk) [inline renderdoc/driver/vulkan/vk_common.h]: nitpick: this isn't in alphabetical order.

File: renderdoc/driver/vulkan/vk_stringise.cpp [hunk 2]
```diff
@@ -789,6 +789,20 @@ std::string DoStringise(const VkCullModeFlagBits &el)
   END_ENUM_STRINGISE();
 }
 
+template <>
+std::string DoStringise(const VkColorComponentFlagBits &el)
+{
+  // technically a bitfield but each combination has a particular meaning
```
Comment (baldurk) [inline renderdoc/driver/vulkan/vk_stringise.cpp]: This comment no longer applies to this enum, since it is a bitfield.

**Comment by baldurk** [comment]
This looks fine, thanks. I've noted a couple of very minor nitpicks.

**Comment by baldurk** [comment]
FYI the failure on travis CI is due to the second commit message not following the [CONTRIBUTING guidelines](https://github.com/baldurk/renderdoc/blob/v1.x/docs/CONTRIBUTING/Preparing-Commits.md#commit-messages) - please reduce it to a 72 character first line summary then any extra information after that.

**Comment by azhirnov** [comment]
I unite commits into one, because:
> Commit messages like "fix stuff" or "compile fix" or any message with "WIP", "amend", "temp", etc should probably not remain in the final PR.

**Pull request merged**